templates/example_cell_clustering.ipynb

```diff
@@ -11,13 +11,15 @@
    "cell_type": "markdown",
```

Comment:
<p>I think the sentence "This data will be needed to compute the average channel expression per cell cluster (the data stored in&nbsp;<code>cell_som_cluster_channel_avg_name</code>&nbsp;and&nbsp;<code>cell_meta_cluster_channel_avg_name</code>), which will be used when computing weighted channel average per cell cluster later on" is confusing. Should probably just be "This data will be needed to compute the <strong>weighted </strong>average channel expression per cell cluster (the data stored in&nbsp;<code>cell_som_cluster_channel_avg_name</code>&nbsp;and&nbsp;<code>cell_meta_cluster_channel_avg_name</code>)" - remove second part of the sentence.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/angelolab/ark-analysis/pull/460/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='12'/>

Reply:
<p>Yeah big sentence barf, think I tripped over myself three times writing this one and it still came out wrong 😅.</p>